### PR TITLE
Added guard-jekyll-plus plugin for smarter Jekyll builds

### PR DESCRIPTION
**Pull request by red2k18**
Below are the features of Guard Jekyll Plus plugin:

1. Changing static files won't trigger a jekyll build
2. Batched processing
3. Reads options from your YAML config file(s)
4. Supports multiple config files
5. Serve with Jekyll or Rack
6. Clear and colorized output.

### CODE REVIEW

**Comment by red2k18** [comment]
Here [https://github.com/imathis/guard-jekyll-plus](url) is the README.md of the guard-jekyll-plus plugin itself. PTAL.

**Comment by blueandgold** [comment]
Can you please add the speed improvement to the PR description?  I am interested to know if we got the same improvement as in the article.

**Comment by red2k18** [comment]
Build time is about the same after adding this plugin as in our case it is proportional to the copies of documentation we maintain on the website.

File: Guardfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# The Guardfile
+# Taken directly from https://github.com/imathis/guard-jekyll-plus
+
+ignore /^_site/ # NOTE: this can interfere with Guard::LiveReload
+
+guard "jekyll-plus", :serve => true do
+  watch /.*/
+end
```
Comment (blueandgold) [inline Guardfile]: add a newline here

File: _config.yml [hunk 2]
```diff
@@ -26,9 +26,10 @@ exclude: ["Gemfile",
           ".travis.yml",
           "travis/",
           "scripts/",
-          ".gitignore",
+          ".git:ignore",
```
Comment (blueandgold) [inline _config.yml]: this is correct?

File: Guardfile (hunk 1 above)
Comment (red2k18) [reply Guardfile]: Done

File: _config.yml (hunk 2 above)
Comment (red2k18) [reply _config.yml]: Fixed it

**Comment by red2k18** [comment]
Closing this. Open #3546 instead.

**Pull request closed without merging**